packages/fiori/src/SearchItem.ts

```diff
@@ -79,6 +91,15 @@ class SearchItem extends ListItemBase {
 	@property()
 	highlightText = "";
 
+	/**
+	 * **Note:** While the slot allows option for setting custom avatar, to match the
```

Comment:
While the slot allows the option of setting a custom avatar, to comply with the design guidelines, use the `ui5-avatar` with size - XS.